structslop.go

```diff
@@ -119,13 +136,59 @@ func run(pass *analysis.Pass) (interface{}, error) {
 			}
 		}
 
+		dtyp := dec.Dst.Nodes[atyp].(*dst.StructType)
+		fields := make([]*dst.Field, 0, len(r.optIdx))
+		dummy := &dst.Field{}
+		for _, f := range dtyp.Fields.List {
+			fields = append(fields, f)
+			if len(f.Names) > 1 {
```

Comment:
Let’s reduce this nesting by inverting:
if len(f.Names) <= 1 {
    continue
}

Reply:
Maybe:

```
		for _, f := range dtyp.Fields.List {
			fields = append(fields, f)
			if len(f.Names) == 0 {
				continue
			}
			for range f.Names[1:] {
				fields = append(fields, dummy)
			}
		}
```

I tend to avoid `<=` or `>=`.